train.py

```diff
@@ -0,0 +1,165 @@
+import torch
+import torch.nn as nn
+from torch.autograd import Variable
+from torch.nn import functional as f
+import os
+import argparse
+from models import DnCNN, PatchLoss, WeightedPatchLoss
+from dataset import RootDataset, get_all_histograms, get_bin_weights, add_noise
+import glob
+import torch.optim as optim
+import uproot
+#from tensorboardX import SummaryWriter
+import matplotlib.pyplot as plt
+from magiconfig import ArgumentParser, MagiConfigOptions, ArgumentDefaultsRawHelpFormatter
+from configs import configs as c
+from torch.utils.data import DataLoader
+import math
+import numpy as np
+
+os.environ["CUDA_DEVICE_ORDER"] = "PCI_BUS_ID"
+os.environ["CUDA_VISIBLE_DEVICES"] = "0"
+
+# create and save truth, noisy, and reconstructed data sets and store in text files
+def make_sample_images(args, model):
```

Comment:
can be removed

Reply:
Ohh does this environment variable exist by default on the LPCGPU: "CUDA_VISIBLE_DEVICES = 0"

Reply:
i just meant this particular function, not the parts above.